src/wrapped/wrappedgtkx112.c

```diff
@@ -20,7 +20,7 @@
 #include "gtkclass.h"
 
 const char* gtkx112Name = "libgtk-x11-2.0.so.0";
-static char* libname = NULL;
```

Comment:
this was here because it use to be able to load gtk-1 also. I guess this is obsolete and can be remove completly, especially on 64bits?

Reply:
Yes. `wrappedgdkx112.c` still requires it, but `wrappedgtkx112.c` does not.